shared/data/constants.js

```diff
@@ -48,9 +48,3 @@ let constants = {
         "automattic": 7
     }
 }
-
-if (typeof window === 'undefined' && module && module.exports)  {
-    module.exports = constants
-} else if (typeof window === 'object') {
-    window.constants = constants
-}
```

Comment:
Both the background processes and the UI code were using `window.constants`, even though the UI code could previously access them using Browserify. As of this change, you can only access the constants using require.

See https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/149#discussion_r170289601 and https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/152 for more info.